kratos/utilities/element_size_calculator.cpp

```diff
@@ -319,6 +390,15 @@ double ElementSizeCalculator<3,4>::ProjectedElementSize(const Geometry<Node<3> >
     return Hvel;
 }
 
+// Prism3D6 version
+template<>
+double ElementSizeCalculator<3,6>::ProjectedElementSize(const Geometry<Node<3> > &rGeometry,
+                                                        const array_1d<double,3>& rVelocity){
```

Comment:
```suggestion
double ElementSizeCalculator<3,6>::ProjectedElementSize(
    const Geometry<Node<3> > &rGeometry,
    const array_1d<double,3>& rVelocity)
{
```
(just taking the chance to adapt to the style guide)

Reply:
Also, I see a lot of local variables in CamelCase. This is also not correct according to the style guide. right?

Reply:
done